node.js: leading slash in the path argument of rest-style api (#460)
cap-js/docs: node.js/core-services.md

Use this method to send synchronous requests to a service for execution.

> -  `method` can be a HTTP method, or a name of a custom action or function

Reviewer: ```suggestion
-  `method` can be an HTTP method, or a name of a custom action or function
```

> -  `path` can be an arbitrary URL, starting with a leading `'/'`, it is passed to a service without any modification as a string. 

Reviewer: ```suggestion
-  `path` can be an arbitrary URL, starting with a leading `'/'`, it is passed to a service without any modification as a string
```

srv.delete('/Books',...)  -->  srv.send('DELETE','/Books',...)
```

> Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. Omitting the leading slash or passing a reflected entity definition instead, constructs *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):

Reviewer: ```suggestion
Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent unmodified to a service. Omitting the leading slash, or passing a reflected entity definition instead, constructs *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):
```

> Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. Omitting the leading slash or passing a reflected entity definition instead, leads to construction of *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):

Reviewer 2: ```suggestion
Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. Omitting the leading slash or passing a reflected entity definition instead, constructs *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):
```

Reviewer 2: @renejeglinsky could you please also double check the wording?

> Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. But you can also use these methods as REST-style variants to run queries by omitting the leading slash in the `path` argument, or by passing a reflected entity definition instead. In that case they start constructing *bound* [`cds.ql` query objects](cds-ql), as their [CRUD-style counterparts](#crud-style-api):

Reviewer 2: I think this paragraph is overly complex. Maybe we can simplify but I don't have a good idea atm....

Author: @johannes-vogel could you please check the simplified variant?
